beacon-chain/p2p/encoder/ssz.go

```diff
@@ -16,7 +16,8 @@ import (
 var _ NetworkEncoding = (*SszNetworkEncoder)(nil)
 
 // MaxGossipSize allowed for gossip messages.
-var MaxGossipSize = params.BeaconNetworkConfig().GossipMaxSize // 1 Mib
+var MaxGossipSize = params.BeaconNetworkConfig().GossipMaxSize // 1 Mib.
```

Comment:
Can we use a proper setter method here to modify this ? Rather than setting a global var that we have. It is  cleaner to understand that way and we do not expose a global variable from this package.